packages/labs/ssr-dom-shim/src/index.ts

```diff
@@ -111,10 +111,17 @@ const CustomElementRegistryShim = class CustomElementRegistry {
 
   define(name: string, ctor: CustomHTMLElementConstructor) {
     if (this.__definitions.has(name)) {
-      throw new Error(
-        `Failed to execute 'define' on 'CustomElementRegistry': ` +
-          `the name "${name}" has already been used with this registry`
-      );
+      if (process.env.NODE_ENV === 'development') {
+        console.warn(
+          `'CustomElementRegistry' already has "${name}" defined. ` +
+            `Overriding in development mode.`
```

Comment:
I don't fully grok the "Overriding in development mode." phrase.

I'd make it as clear as possible what the problem is and what will be different in production mode. So say that this is either caused by HMR and is fine, or must be fixed, ideally how to check it (run same page in prod mode?)

Reply:
Yeah good call. It seems better to be a bit more verbose here. Updated!